### PR TITLE
batabases/week2

### PR DESCRIPTION
**Pull request by sofiiadidovych**

### CODE REVIEW

**Review by aina21 — APPROVED** [review]
Good job 👍

File: databases/week2/part3.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+-- Part 3: More queries
```
Comment (aina21) [inline databases/week2/part3.sql:1]: you should use lesson2 data for this part, 
https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql

**Review by aina21 — CHANGES_REQUESTED** [review]

File: databases/week2/part2.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    begins DATE,
+    ends DATE
+);
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+ CREATE TABLE student (
+	id INT PRIMARY KEY AUTO_INCREMENT, 
+    name VARCHAR(15), 
+    email TEXT, 
+    phone INT,
```
[the remark this answers is not in the document]
Comment (sofiiadidovych) [reply databases/week2/part2.sql:17]: Thank you, I will change it to varchar.

File: databases/week2/part3.sql (hunk 1 above)
Comment (sofiiadidovych) [reply databases/week2/part3.sql:1]: I did use this data, but as we used it previously, it was already saved.

**Pull request merged**